.gitignore

```diff
@@ -1,4 +1 @@
-/dist
-/node_modules
-/out
```

Comment:
How does git know to ignore `/out`?

Reply:
out should not exist on root level anymore becau it only exists in /packages/matter.js/out and it is ignored there